src/Http/Controllers/VoyagerController.php

```diff
@@ -63,6 +63,10 @@ public function upload(Request $request)
             } else {
                 $status = __('voyager::media.error_uploading');
             }
+        }else if(in_array($ext, ['pdf','zip', 'doc', 'docx'])){
```

Comment:
Where do these extensions come from?
We have a defined set of extensions for the media-manager. Maybe we can use those?